src/plugins/log/log.js

```diff
@@ -81,9 +113,30 @@ Log.getInstance = function() {
   return this._instance
 }
 
+/**
+ * Specify a `level` to log.
+ * @method setLevel
+ * @param {Number} level valid level log [0,1,2,3,4].
+ */
```

Comment:
I'd describe this as "configure the minimum level to be logged". The `setLevel` only restricts the logs up to a certain value, logging whatever message with a higher log level. So, when set to `LEVEL_INFO` the log would display messages from `LEVEL_INFO` up to `LEVEL_ERROR`.